cypress/integration/SearchPage.feature

```diff
@@ -1,248 +1,217 @@
-    When you click search
-    Then you can see the first page of the results
-    When you click the next page button
-    Then you can see the second page of the results
-    When you click the previous page button
-    Then you can see the first page of the results
-
-  Scenario: Searching using attributes in TEType domain has disabled pagination
-    Given you are on the default search page
-    And you select the search domain Person
-    And you expand the attributes search area
-    And for Person you fill in values that will return less than 5 results
-    When you click search
-    Then you can see the first page of the results
-    And the next page button is disabled
-
-  Scenario: Searching using only date range values as attributes
-    Given you are in the search page with the Adult Woman being preselected from the url
-    When you fill in the date of birth
-    And you click search
-    Then you can see the first page of the results
-
-  Scenario: Searching using zip code range values as attributes
-    Given you are in the search page with the TB program being preselected from the url
-    When you expand the attributes search area
-    And you fill in the zip code range numbers
-    And you click search
-    Then you can see the first page of the results
-
-  Scenario: Searching using zip code range and name values as attributes
-    Given you are in the search page with the TB program being preselected from the url
-    When you expand the attributes search area
-    And you fill in the zip code range numbers
-    And you fill in the first name
-    And you click search
-    Then you can see the first page of the results
-
-  Scenario: Fallback search with results
-    Given you are in the search page with the Child Programme being preselected from the url
-    And you expand the attributes search area
-    When you fill in the first and last name with values that will return results
-    And you click search
-    And you can see the first page of the results
-    When you click the fallback search button
-    Then you stay in the same page with all program results
-
-# TODO - Commenting out this test and creating a new tech-ticket (TECH-866)
-# Fallback-search relies on history-state, which is not suppported with the hash router - needs rewriting
-#  Scenario: Fallback search keeps the form values and the attributes search area is being expanded
-#    Given you are in the search page with the Child Programme being preselected from the url
-#    And you expand the attributes search area
-#    When you fill in the first and last name with values that will return results
-#    And you click search
-#    And you can see the first page of the results
-#    When you click the fallback search button
-#    Then you see the attributes search area being expanded
-#    And that first and last name are prefilled
-
-  Scenario: Fallback search excludes attributes that are not relevant in the tetype scope
```

Comment:
Hey,
Any reason why the scenarios `Fallback search excludes attributes that are not relevant in the tetype scope` and `Fallback search with results and navigation back to the program show` were deleted? They were not disabled beforehand which means that in theory, they are valid tests. 
Thank you!